common/country_definitions/00_countries.txt

```diff
@@ -721,7 +721,11 @@ SWI = {
 	
 	capital = STATE_WEST_SWITZERLAND
 	cultures = { alemannic francoprovencal }
+<<<<<<< HEAD
```

Comment:
Some issues here :) I found many of these.

I'm not a dev but I took a look anyway